Resources/Prototypes/_Funkystation/Store/uplink_catalog.yml

```
@@ -0,0 +1,10 @@
+- type: listing
+  id: UplinkChangelingInfectImplanter
+  name: uplink-changelinginfect-implant-name
+  description: uplink-changelinginfect-implant-desc
+  icon: { sprite: /Textures/Objects/Specific/Medical/syndi_implanter.rsi, state: implanter0 }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)